### ISSUE TITLE
Customizable Rollups: Account Soft Credit Jobs Using Default 200 Batch Size

### ISSUE THREAD

**Issue by jesslopez-sf** [SalesforceFoundation/Cumulus#4166]
If a customer has many rollups/Filter Groups and many Accounts with thousands of Account Soft Credits, they may hit heap size errors when the CRLP_Account_AccSoftCredit_BATCH and CRLP_AccountSkew_AccSoftCredit_BATCH run. Typically we would suggest reducing the batch size, but we don't have one for this job. The suggestion is to either create its own or share the Rollups_Account_Soft_Credit_Batch_Size__c field (for non-skew at least).

Either solution should be set in: [CRLP_RollupBatch_SVC.cls, Lines 265-296](https://github.com/SalesforceFoundation/Cumulus/blob/master/src/classes/CRLP_RollupBatch_SVC.cls#L265-L296)

**Sample Stack Trace:**
```
Class.npsp.CMT_FilterRule.isIncluded: line 149, column 1
Class.npsp.CMT_FilterRuleEvaluation_SVC.evaluateFilterGroup: line 115, column 1
Class.npsp.CRLP_RollupAccSoftCredit_SVC.processRollups: line 80, column 1
Class.npsp.CRLP_RollupProcessor_SVC.completeRollupForParent: line 303, column 1
lass.npsp.CRLP_RollupProcessor_SVC.startRollupProcess: line 115, column 1
Class.npsp.CRLP_Batch_Base_NonSkew.execute: line 271, column 1

```

See Work Item for full Investigation.

**Comment by jesslopez-sf** [comment]
**lurch: add

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000008LmKlQAK" target="_blank">W-032439</a>

**Comment by mrbelvedere** [comment]
Included in beta release 3.154 (Beta 6)

**Comment by mrbelvedere** [comment]
Included in production release 3.154
